network/config.go

```diff
@@ -36,17 +36,16 @@ type AddrAndBalance struct {
 	Balance uint64
 }
 
-// Backend enumerator type
+// The type of network runner
```

Comment:
Shouldn't comments of exported items start with the exported name?

Reply:
I'm not super committed to that approach -- sometimes I think it makes the comment sound clunky. In any case, changed this comment to start with `Backend`